website/pages/docs/deployment/ecs.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Running CloudQuery on Amazon ECS
+tag: tutorial
+date: 2023/03/03
+---
+
+# Deploy with Amazon ECS
+
+In this tutorial, we will show you how to load AWS resources into Amazon S3 by running CloudQuery as a service in [Amazon ECS](https://github.com/features/actions), using the AWS source- and S3 destination plugins.
+
+## Prerequisites
+
+1. S3 Bucket that will hold the data. If the bucket is in a different account than where you will be deploying the ECS service then you will need to ensure that there is a bucket policy that allows for cross account access.
+
+
+## Creating the CloudQuery configuration file
+
+Under the root of your repository, create a new `cloudquery.yml` file with the following content:
+
+```yaml copy
+kind: source
+spec:
+  name: 'aws'
+  path: cloudquery/aws
+  version: "VERSION_SOURCE_AWS"
+  destinations: ['s3']
+---
+kind: destination
+spec:
+  name: 's3'
+  path: cloudquery/s3
+  version: "VERSION_DESTINATION_S3"
+  spec:
+    bucket: <DESTINATION_BUCKET_NAME>
+```
+
+You will need to replace `<DESTINATION_BUCKET_NAME>` with the name of the S3 Bucket where you want to store the data. 
+
+
+> For more configuration options, [visit our docs](/docs/reference/source-spec)
+
+## Solution Overview
+
+The Cloudformation will stand up a number of resources including:
+- ***ECS Service and Task definition***: This uses the base image that the [CloudQuery Team maintains](https://github.com/cloudquery/cloudquery/pkgs/container/cloudquery). On top of that image this solution provides an alternative entrypoint that enables users to specify the configuration file as a base64 encoded string as an environment variable. This allows users to not have to build and maintain a custom image.
+- ***Eventbridge Schedule***: Th
```

Comment:
```suggestion
- ***EventBridge Schedule***: Th
```

Reply:
There's something more missing here it seems (currently the text just says `Th`)